openmessaging-runtime/src/main/java/io/openmessaging/connect/runtime/service/RebalanceImpl.java

```diff
@@ -76,44 +76,24 @@ public void doRebalance() {
         Map<String, List<ConnectKeyValue>> curTaskConfigs = configManagementService.getTaskConfigs();
 
         ConnAndTaskConfigs allocateResult = allocateConnAndTaskStrategy.allocate(curAliveWorkers.keySet(), worker.getWorkerId(), curConnectorConfigs, curTaskConfigs);
-        log.info("allocated connector:"+ allocateResult.getConnectorConfigs());
-        log.info("allocated task:"+ allocateResult.getTaskConfigs());
+        log.info("allocated connector:" + allocateResult.getConnectorConfigs());
```

Comment:
We recommend the first letter in uppercase, and use placeholders({}) in the log instead "+".